spec/ruby-push-notifications/mpns/mpns_connection_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+module RubyPushNotifications
+  module MPNS
+    describe MPNSConnection do
+
+      describe '::post' do
+
+        let(:body) { 'abc' }
+        let(:headers) {
+          {
+            'x-notificationstatus' => ['Received'],
```

Comment:
No I don't ether.. I thinked to pass 
````
responses << { device_url: url.to_s, result: response  }
````
instead
https://github.com/lazaronixon/ruby-push-notifications/blob/mpns/lib/ruby-push-notifications/mpns/mpns_connection.rb#L51